### PR TITLE
Grammatical errors fixed in CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by Apoo711**
### Type

- [ ] Bugfix
- [x] Enhancement

- [ ] New feature

### Description of the changes
I fixed a couple of grammatical errors in contributing.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
 
 ### 1.1 Why do these guidelines exist?
-Red is an open source project. This means that each and every one of the developers and contributors who have helped make Red what it is today have done so by volunteering their time and effort. It takes a lot of time to coordinate and organize issues and new features and to review and test pull requests. By following these guidelines you will help the developers streamline the contribution process and save them time. In doing so we hope to get back to each and every issue and pull request in a timely manner.
+Red is an open-source project. This means that each and every one of the developers and contributors who have helped make Red what it is today have done so by volunteering their time and effort. It takes a lot of time to coordinate and organize issues and new features and to review and test pull requests. By following these guidelines you will help the developers streamline the contribution process and save them time. In doing so we hope to get back to each and every issue and pull request in a timely manner.
```
Comment (Kreusada) [inline CONTRIBUTING.md]: There is no need for this change.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,13 +23,13 @@
...
 
 ### 1.2 What kinds of contributions are we looking for?
-We love receiving contributions from our community. Any assistance you can provide with regards to bug fixes, feature enhancements, and documentation is more than welcome.
+We love receiving contributions from our community. Any assistance you can provide with regard to bug fixes, feature enhancements, and documentation is more than welcome.
```
Comment (Kreusada) [inline CONTRIBUTING.md]: There was no need for this change, `regards` sounds much better

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,13 +23,13 @@
...
 
 # 2. Ground Rules
-1. Ensure cross compatibility for Windows, Mac OS and Linux.
+1. Ensure cross-compatibility for Windows, Mac OS, and Linux.
```
Comment (Kreusada) [inline CONTRIBUTING.md]: There shouldn't be a comma there, because `and` comes right after

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -40,18 +40,18 @@ We love receiving contributions from our community. Any assistance you can provi
 Unsure of how to get started contributing to Red? Please take a look at the Issues section of this repo and sort by the following labels:
 
 * beginner - issues that can normally be fixed in just a few lines of code and maybe a test or two.
-* help-wanted - issues that are currently unassigned to anyone and may be a bit more involved/complex than issues tagged with beginner.
+* help-wanted - issues that are currently unassigned to anyone and maybe a bit more involved/complex than issues tagged with beginner.
```
Comment (Kreusada) [inline CONTRIBUTING.md:73]: `maybe` and `may be` mustn't be confused with one another. One change you could make would be to add `are` before the `maybe`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -40,18 +40,18 @@ We love receiving contributions from our community. Any assistance you can provi
...
-At this point you're ready to start making changes. Feel free to ask for help; everyone was a beginner at some point!
+At this point, you're ready to start making changes. Feel free to ask for help; everyone was a beginner at some point!
 
 # 4. Getting Started
 
 Red's repository is configured to follow a particular development workflow, using various reputable tools. We kindly ask that you stick to this workflow when contributing to Red, by following the guides below. This will help you to easily produce quality code, identify errors early, and streamline the code review process.
 
 ### 4.1 Setting up your development environment
-The following requirements must be installed prior to setting up:
+The following requirements must be installed before setting up:
```
Comment (Kreusada) [inline CONTRIBUTING.md]: What was wrong with `prior to`?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -134,10 +134,10 @@ If you find yourself wanting a feature that Red does not already have, you're pr
 
 We have a core team working tirelessly to implement new features and fix bugs for the Red community. This core team looks at and evaluates new issues and PRs on a daily basis.
 
-The decisions we make are based on a simple majority of that team or by decree of the project owner.
+The decisions we make are based on a simple majority of that team or by the decree of the project owner.
 
 ### 5.1 Issues
-Any new issues will be looked at and evaluated for validity of a bug or for the usefulness of a suggested feature. If we have questions about your issue we will get back as soon as we can (usually in a day or two) and will try to make a decision within a week.
+Any new issues will be looked at and evaluated for the validity of a bug or the usefulness of a suggested feature. If we have questions about your issue we will get back as soon as we can (usually in a day or two) and will try to decide within a week.
```
Comment (Kreusada) [inline CONTRIBUTING.md]:
```suggestion
Any new issues will be looked at and evaluated for the validity of a bug or for the usefulness of a suggested feature. If we have questions about your issue we will get back as soon as we can (usually in a day or two) and will try to decide within a week.
```

**Review by Kreusada — CHANGES_REQUESTED** [review]
This PR caught my eye with a few changes which I didn't agree with, see the review comments for more info

**Comment by Apoo711** [comment]
> This PR caught my eye with a few changes which I didn't agree with, see the review comments for more info

Ok, thank you for pointing out the mistakes, I'll fix them.

File: CONTRIBUTING.md (hunk 4 above)
Comment (Apoo711) [reply CONTRIBUTING.md:73]: Should i change it to "and are may be" or should i keep the maybe?

**Comment by Apoo711** [comment]
Added the fixes that Kreusada suggested, also added more fixes.

**Comment by Apoo711** [comment]
I recommitted the changes because no changes went through before.

**Comment by Drapersniper** [comment]
Imma say a lot of these changes are unnecessary, and there's a change in tone in a lot of your changes. 

This is a official one-pager document and the intended tone formal English.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,159 +1,243 @@
 # Contents
```
Comment (Drapersniper) [inline CONTRIBUTING.md:1]: What is the purpose of all these extra empty lines?

File: CONTRIBUTING.md (hunk 4 above)
Comment (Drapersniper) [reply CONTRIBUTING.md:73]: You should keep at is easy as it was grammatically correct

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,159 +1,243 @@
...
 
 * beginner - issues that can normally be fixed in just a few lines of code and maybe a test or two.
-* help-wanted - issues that are currently unassigned to anyone and may be a bit more involved/complex than issues tagged with beginner.
+
+* help-wanted - issues that are currently unassigned to anyone and maybe a bit more involved/complex than issues tagged with beginner.
 
 **Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
 
-At this point you're ready to start making changes. Feel free to ask for help; everyone was a beginner at some point!
+At this point, you’re ready to make changes. Feel free to ask for help; everyone was a beginner at some point!
```
Comment (Drapersniper) [inline CONTRIBUTING.md:77]: It's an action there for "you are ready to start making making changes" is grammatically correct with a formal undertone.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,159 +1,243 @@
...
 Currently, tox does the following, creating its own virtual environments for each stage:
+
 - Runs all of our unit tests with [pytest](https://github.com/pytest-dev/pytest) on python 3.8 (test environment `py38`)
+
 - Ensures documentation builds without warnings, and all hyperlinks have a valid destination (test environment `docs`)
+
 - Ensures that the code meets our style guide with [black](https://github.com/psf/black) (test environment `style`)
 
-To run all of these tests, just run the command `tox` in the project directory.
+To run these tests, just run the command `tox` in the project directory.
```
Comment (Drapersniper) [inline CONTRIBUTING.md:145]: These two sentences have two different meanings, you should Reinstate the meaning of the first which is to run all tests and not a subsection.

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,159 +1,243 @@
...
 
 To run a subset of these tests, use the command `tox -e <env>`, where `<env>` is the test environment you want tox to run. The test environments are noted in the dot points above.
 
-Your PR will not be merged until all of these tests pass.
+Your PR will not be merged until these tests pass.
```
Comment (Drapersniper) [inline CONTRIBUTING.md:149]: Same as  above, all these tests are a requirements so revert this as this statement is incorrect

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,159 +1,243 @@
...
 ### 4.7 How To Report A Bug
+
 Please see our **ISSUES.MD** for more information.
 
 ### 4.8 How To Suggest A Feature Or Enhancement
+
 The goal of Red is to be as useful to as many people as possible, this means that all features must be useful to anyone and any server that uses Red.
 
-If you find yourself wanting a feature that Red does not already have, you're probably not alone. There's bound to be a great number of users out there needing the same thing and a lot of the features that Red has today have been added because of the needs of our users. Open an issue on our issues list and describe the feature you would like to see, how you would use it, how it should work, and why it would be useful to the Red community as a whole.
+If you want a feature that Red does not already have, you’re probably not alone. There’s bound to be a great number of users out there needing the same thing, and a lot of the features that Red has today have been added because of the needs of our users. Open an issue on our issues list and describe the feature you would like to see, how you would use it, how it should work, and why it would be useful to the Red community.
```
Comment (Drapersniper) [inline CONTRIBUTING.md:205]: Once again, the change of tone here is not necessary.

Comment (Drapersniper) [reply CONTRIBUTING.md:205]: And the last sentence that removal was also not necessary

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -1,159 +1,243 @@
...
 
 # 5. Code Review Process
 
 We have a core team working tirelessly to implement new features and fix bugs for the Red community. This core team looks at and evaluates new issues and PRs on a daily basis.
```
Comment (Drapersniper) [inline CONTRIBUTING.md:210]: The issue unneeded here

**Review by Drapersniper — CHANGES_REQUESTED** [review]

**Comment by Apoo711** [comment]
Thank you draper! I''l be working on all of things you pointed out.

**Pull request closed without merging**